configs/nextflow.base.config

```diff
@@ -1,36 +1,43 @@
+workDir = "/filepath/to/workdir" //edit
+
 params {
+	 // FREQUENTLY CHANGED PARAMETERS //
+
 	root = "/filepath/to/jasen" //edit
-	// DATABASES //
+	queue = ""
+	outdir = "/filepath/to/outdir" //edit
+	krakenDb = "/filepath/to/kraken_db" //edit if useKraken = true
+	useHostile = false
+	useKraken = true
+	useSkesa = true // Use Spades by setting useSkesa = false
+	scratch = true 
+	cronCopy = false
+	devMode = false
+
+	// ASSET PROVIDED DATABASES //
+
 	amrfinderDb = "${root}/assets/amrfinder_db/latest"
 	resfinderDb = "${root}/assets/resfinder_db"
 	pointfinderDb = "${root}/assets/pointfinder_db"
 	serotypefinderDb = "${root}/assets/serotypefinder_db"
 	virulencefinderDb = "${root}/assets/virulencefinder_db"
 	mlstBlastDb = "${root}/assets/mlst_db/blast"
 	pubMlstDb = "${root}/assets/mlst_db/pubmlst"
-	krakenDb = "/filepath/to/kraken_db" //edit if useKraken = true
 	hostileIdx = "${root}/assets/hostile_db/human-t2t-hla"
-	// INPUT & OUTPUT //
-	publishDir = ""
+
+	// STANDARDIZED PARAMETERS //
+
+	containerDir = "${root}/container"
+	workDir = "${workDir}"
```

Comment:
How sure are you that this works?

Reply:
Yea workdir isn't applied unless its under base.
L31 could be removed. I have no strong feelings about it. Just added it for backwards compatibility.